godtools/App/Services/Renderer/BackgroundImageRenderer/MobileContentBackgroundImageRenderer.swift

```diff
@@ -42,41 +43,50 @@ class MobileContentBackgroundImageRenderer: MobileContentBackgroundImageRenderer
         
         case .fill:
             return scaleRectToFillContainer(container: container, rect: rect)
-        
-        case .fillHorizontally:
+                    
+        case .fillX:
             return scaleRectToHorizontallyFitContainer(container: container, rect: rect)
         
-        case .fillVertically:
+        case .fillY:
             return scaleRectToVerticallyFitContainer(container: container, rect: rect)
+            
+        default:
+            return scaleRectToFitContainer(container: container, rect: rect)
```

Comment:
since your default case is the same as the `.fit` case, can you combine them like this:

```swift
case .fit:
default:
    return scaleRectToFitContainer(container: container, rect: rect)
```

that will make it more clear you are defaulting to the fit scale type if the provided scaleType isn't found

Reply:
@frett unfortunately it appears I can't do that with the default case.

Reply:
k, wasn't sure language limitations in Swift. I actually can't do that in Kotlin, but it works fine in Java. it depends on how the language handles switches

Reply:
I can chain cases by comma separating them, but it won't allow me to do it with the default case.  I wish I didn't have to include the default case, but the compiler is forcing me too.  I'm pretty sure if new cases are added here I won't get a compiler error because the default is there.

Reply:
yeah, it would be nice if the compiler recognized it as an enumeration and that you were handling all the cases already

Reply:
this looks like a feature request for this functionality:
https://youtrack.jetbrains.com/issue/KT-48068

I starred and liked it so that I get notifications if the status ever changes